create-auth.c

```diff
@@ -341,6 +347,10 @@ int main(int argc, char **argv)
     /* Initialize timestamp to current time (in UTC). */
     time(&t);
     tm = gmtime(&t);
+    if (!tm) {
+         printf("gmtime() failed: %d, %s\n", errno, strerror(errno));
```

Comment:
this and above could've used `perror`, but this is fine too

Reply:
I usually prefer not to use perror() in these cases as I find it useful to have the raw errno as well to avoid needing to try and map the string back to an errno constant. Especially useful when strerror() localized in a different language (though not relevant in this case).